lib/literal/data.rb

```diff
@@ -9,7 +9,7 @@ def prop(name, type, kind = :keyword, reader: :public, predicate: false, default
 		def literal_properties
 			return @literal_properties if defined?(@literal_properties)
 
-			if superclass.is_a?(Literal::Data)
+			if superclass < Literal::Data
```

Comment:
alternatively we could say:
```suggestion
			if superclass.is_a?(Literal::Properties)
```
not sure why it needs to check for `Data` here anyway

Reply:
Yeah, this should probably be this, or even `if Literal::Properties === superclass`.